daemon/graphdriver/zfs/zfs_linux.go

```diff
@@ -15,7 +15,7 @@ func checkRootdirFs(rootdir string) error {
 	}
 
 	if graphdriver.FsMagic(buf.Type) != graphdriver.FsMagicZfs {
-		logrus.Debugf("[zfs] no zfs dataset found for rootdir '%s'", rootdir)
+		logrus.Debugf("[zfs] no zfs dataset found for rootdir '%s', found '%s' filesystem", rootdir, buf.Type)
```

Comment:
`buf.Type` is an `int64`, so probably isn't very useful to print; perhaps this should use a similar approach as is used by the other drivers? https://github.com/moby/moby/blob/90fbe6a7373da81b36d1427e0c2764c0b583f421/daemon/graphdriver/aufs/aufs.go#L91-L103